yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -764,6 +653,17 @@ export class L2Block {
   }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 640]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This should be moved into the `tx_l2_logs` such that we can just call `logs.hash()` should make the full calldata computation much cleaner.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4541?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->